lang/python/python/Makefile

```diff
@@ -194,17 +194,34 @@ define Build/Compile
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib $(1)/usr/lib/pkgconfig
```

Comment:
nitpick: splitting  `$(INSTALL_DIR)` into multiple lines is mostly cosmetic; doesn't add much value
personally, i like having all the `$(INSTALL_DIR)` in a single place, since it's fewer lines to look at;

Reply:
When there are 3 directories or less I think having one line is fine; when there are more, I think it helps to group `$(INSTALL_DIR)`s with their `$(CP)`s (to make sure no directories are missed).

Can change this back if you prefer :smile:

Reply:
When there are 3 directories or less I think having one line is fine; when there are more, I think it helps to group `$(INSTALL_DIR)`s with their `$(CP)`s (to make sure no directories are missed).

Can change this back if you prefer :smile:

Reply:
I've changed this back to one line :joy: